samples/__snapshots__/filters.js

```diff
@@ -0,0 +1,501 @@
+exports['filters should filter with row regex 1'] = `
```

Comment:
I hate this question, but do we need license headers on snapshots @bcoe?

Reply:
I advised @billyjacobson against this, the `__snapshots_` are generated automatically using the command `npm run test:snap`; I think it's fine to ignore the license header check in this PR, but we should add the ignore rule to `synthtool`'s templates.